github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRPullRequest.java

```diff
@@ -60,7 +62,7 @@ public GitHubPRPullRequest(GHPullRequest pr) throws IOException {
         userLogin = pr.getUser().getLogin();
         number = pr.getNumber();
         prUpdatedAt = pr.getUpdatedAt();
-        issueUpdatedAt = pr.getIssueUpdatedAt();
+        issueUpdatedAt = pr.getUpdatedAt();
```

Comment:
Maybe even should be changed to `pr.getIssue(pr.getNumber()).getUpdatedAt()` need to be tested